CMSIS/NN/Source/PoolingFunctions/arm_avgpool_s8.c

```diff
@@ -31,6 +31,36 @@
 #include "arm_math.h"
 #include "arm_nnfunctions.h"
```

Comment:
Please update the revision/date and also add a 'Yes' in the readme  https://github.com/ARM-software/CMSIS_5/pull/851/commits/f67c89e03859ba0407439b832b6b8a84384ab123#diff-f03fd8da949824639265e9dc65de41dc

Reply:
Yes